pages/album/[id].vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts" setup>
+const { t } = useI18n();
+toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.album"));
+
+const { id } = useRoute().params;
+const albumId = Number(id);
+
+const { data: album } = useAlbum({ id: albumId });
```

Comment:
When now navigating from one album to another, the view won't update. We've had this before in #60 with playlists.

Reply:
I think it might work to wrap the `Number(id)` in a `computed`, such as this:

```ts
const albumId = computed(() => Number(id))
```

Reply:
I highly doubt this, because `id` is of type `string`. Having a string inside a `computed` function won't do anything, it must have a form of reactiveness, like `Ref<T>`.

What I still need to wrap my head around is, that nuxt changes something on the routing so navigating to the same route triggers a re-rendering of the page; that said, my changes from #60 are not required anymore, but we are constantly re-rendering components even though we don't need it ... weird ...